File: timesTables.R
tab <- 0
range <- 0
tab <- as.numeric(readline("What table do you want to see:"))
range <- as.numeric(readline("What number do you want to see it up to:"))

for (i in seq(1, range)){
  result <- tab * i
  print(paste("The times table by", i, "results in", result, sep = " "))
}

File: TimesTables.java

public class TimesTables {

	public void createTable(int table, int range){
		System.out.println("Times Tables for " + table + " for the range of " + range);
		for(int r = 1; r <= range; r++){
			System.out.println(table + "*" + r + "=" + table * r);
		}
	}
	
}
